Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -44,6 +43,23 @@ public IEnumerable ItemsSource
 			set => SetValue(ItemsSourceProperty, value);
 		}
 
+		public static readonly BindableProperty RemainingItemsThresholdReachedCommandProperty = BindableProperty.Create(nameof(RemainingItemsThresholdReachedCommand), typeof(ICommand), typeof(ItemsView), null);
```

Comment:
`null` is probably redundant here, y/n?

Reply:
no, it's fine. defaultValue parameter shouldn't have been made optional.

Reply:
a `Command`, but no `CommandParameter` ?

Reply:
What would a typical `CommandParameter` look like in this case?